packages/block-editor/src/components/border-radius-control/style.scss

```diff
@@ -1,26 +1,13 @@
+.components-border-radius-control__header {
```

Comment:
Should also give the radius range control marks the same style as the spacing one? White horizontal marks instead of grey.

https://github.com/WordPress/gutenberg/blob/bd11946901d4ca837048132c14cef65ae8c28574/packages/block-editor/src/components/spacing-sizes-control/style.scss#L25-L25